packages/axe-core-rspec/README.md

```diff
@@ -189,11 +189,11 @@ All of the described clauses may be mixed and matched with method chaining. Belo
 ```rb
 expect(page).to be_axe_clean.within('.main', '.header').excluding('.footer')
 
-expect(page).to be_axe_clean.excluding('#sidebar').according_to(:wcag2a, :wcag2aa).skipping(:color-contrast)
```

Comment:
For what it's worth, we had `:color_contrast` in our code and that worked until we upgraded from `4.2.1` to `4.3.0`.